r/tools/nixlibs.R

```diff
@@ -462,27 +471,41 @@ turn_off_all_optional_features <- function(env_var_list) {
   replace(env_var_list, names(turn_off), turn_off)
 }
 
+get_component_names <- function() {
+  if (!isTRUE(Sys.which("bash") != "")) {
+    stop("nixlibs.R requires bash to be installed and available in your PATH")
+  }
+  deps_bash <- normalizePath("tools/download_dependencies_R.sh", mustWork = TRUE)
```

Comment:
Do you need to normalizePath? Relative path should be fine here

Reply:
Good point - I was using it as a check that the file exists, but bash error if it doesn't.